assignments/P2.md

```diff
@@ -12,16 +12,16 @@ Solve a puzzle and uncover the secret message by demonstrating your knowledge of
 
 #### What to submit:
 
-* Patch 1 adds `username/P2/ctf.c` and `username/P2/Makefile`, each as unmodified copies of `P2/ctf.c` and `P2/Makfefile` respectively
```

Comment:
putting the module into a subdir doesn't make sense because it needs to reference constants.h which will be in parent directory. I think that the solution program is more similar to the testing program in the other assignments and it is what should go into its own folder.